### PR TITLE
Fis 123 create a structure

### PR DESCRIPTION
**Pull request by hrvojevincek**
Redesign of Dashboard.
Created components and structure so we can import Panels.
Buttons are up for just testing purposes. 
Page is functional.

H

### CODE REVIEW

**Comment by craigostrin** [comment]
I'd rename `DashboardPanel` to `DashboardContent`, and pls remove the placeholder stuff so others can add things to the page without creating conflicts

File: apps/client/components/ui/Dashboard/DashboardContent.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <NominationBox
+            users={users}
+            loggedUserId={loggedUserId}
+            report={report}
+            cycleId={cycleId}
+          ></NominationBox>
+          <Panel>
+            <PanelHeader>NOMINATION PHASE</PanelHeader>
+            <PanelContent>
+              {users.map((user) => (
```
Comment (craigostrin) [inline apps/client/components/ui/Dashboard/DashboardContent.tsx:25]: can you remove this? there shouldn't be any placeholder content when this gets merged

File: apps/client/components/ui/Dashboard/DashboardContent.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  users,
+  cycleId,
+  loggedUserId,
+  report,
+}) {
+  return (
+    <div className="mt-10 grid grid-cols-3 max-w-screen h-[34rem] gap-10">
+      <div className="col-span-2 h-full grid grid-col-2 gap-10 max-w-screen">
+        <div className="border-2 grid-rows-1 border-black">
+          <NominationBox
```
Comment (craigostrin) [inline apps/client/components/ui/Dashboard/DashboardContent.tsx:16]: remove nominationbox, why is it here

File: apps/client/components/ui/Dashboard/DashboardTop.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import Photo from '../photo'
+
+function DashboardTop({ firstName, lastName, title, photo }) {
+  return (
+    <div className="bg-neutral-100 justify-between items-center h-24 grid grid-cols-6">
+      <h1 className="text-3xl text-green-mediumgreen font-extrabold col-span-4">
+        Team member Panel
```
Comment (craigostrin) [inline apps/client/components/ui/Dashboard/DashboardTop.tsx:7]: this should be dynamic too, no? so we could use the Dashboard Top for the User Dashboard and for the Manager Dashboard

File: apps/client/components/ui/Dashboard/NominationPhase.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { Panel, PanelContent, PanelHeader } from '../Panel'
```
Comment (craigostrin) [inline apps/client/components/ui/Dashboard/NominationPhase.tsx:1]: yo, just a heads up that I'm currently working on the NominationPanelContent.

this doesn't conflict so far, but I want to make sure you're aware :)

File: apps/client/tailwind.config.js [hunk 5]
```diff
@@ -33,6 +33,7 @@ module.exports = {
         },
         green: {
           light: 'hsl(var(--light-green))',
+          mediumgreen: 'hsl(var(--medium-green))',
```
Comment (craigostrin) [inline apps/client/tailwind.config.js:36]: this should just be called 'medium'

otherwise you're gonna have to write 'text-green-mediumgreen' lol

**Review by olive20xx — CHANGES_REQUESTED** [review]

File: apps/client/components/ui/Dashboard/DashboardTop.tsx (hunk 3 above)
Comment (hrvojevincek) [reply apps/client/components/ui/Dashboard/DashboardTop.tsx:7]: thats the next step

File: apps/client/components/ui/Dashboard/NominationPhase.tsx (hunk 4 above)
Comment (hrvojevincek) [reply apps/client/components/ui/Dashboard/NominationPhase.tsx:1]: ok ill remove

File: apps/client/tailwind.config.js (hunk 5 above)
Comment (hrvojevincek) [reply apps/client/tailwind.config.js:36]: change it

Comment (craigostrin) [reply apps/client/tailwind.config.js:36]: k

**Review by olive20xx — APPROVED** [review]

**Pull request merged**